### PR TITLE
Migrate `PatientSearchResultsScreen` to a fragment

### PR DESCRIPTION
**Pull request by vinaysshenoy**

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsScreen.kt [hunk 1]
```diff
@@ -114,20 +116,16 @@ class PatientSearchResultsScreen :
...
+    super.onViewCreated(view, savedInstanceState)
     setupScreen()
+    searchResultsView.searchWithCriteria(screenKey.criteria)
   }
 
   override fun onAttachedToWindow() {
     super.onAttachedToWindow()
     delegate.start()
-    searchResultsView.searchWithCriteria(screenKey.criteria)
+
```
Comment (janhavisinghh) [inline app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsScreen.kt]: Can we remove this extra line please?

**Review by janhavisinghh — CHANGES_REQUESTED** [review]
LGTM. Left one comment PTAL and can you also rebase your branch while at it?

File: app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsScreen.kt (hunk 1 above)
Comment (vinaysshenoy) [reply app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsScreen.kt]: This line was removed in a later commit.

Comment (janhavisinghh) [reply app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsScreen.kt]: Oh sorry, missed it.

**Review by janhavisinghh — APPROVED** [review]

**Pull request merged**